cfg/globals.go

```diff
@@ -3,4 +3,8 @@ package cfg
 import "time"
 
 // RequestTimeout is the maximum duration to allow an API request to execute
-const RequestTimeout = 2 * time.Minute
+const (
+	RequestTimeout = 2 * time.Minute
+
+	IndexerTaskEnabled = false
```

Comment:
Is there a reason for this to be a global setting instead of just a default for a property of the Config object?

Reply:
golang circular dependency cause problems.

Reply:
i was having issues with golang cycle dependencies, but i guess it was something else..